src/ctypes/dune

```
@@ -0,0 +1,42 @@
+(rule
+ (with-stdout-to
+  ctypes_primitives.ml
+  (run ../configure/gen_c_primitives.exe)))
+
+(rule
+ (deps
+  (:header %{lib:integers:ocaml_integers.h}))
+ (target ocaml_integers.h)
+ (action
+  (copy %{header} %{target})))
+
+(library
+ (name ctypes)
+ (public_name ctypes)
+ (wrapped false)
+ (libraries bytes integers bigarray-compat)
```

I believe this should be removed after https://github.com/ocaml/ocaml/pull/11007#discussion_r896058677 for OCaml 5.0 compatibility.
```suggestion
 (libraries integers bigarray-compat)
```